### ISSUE TITLE
Document outline view?

### ISSUE THREAD

**Issue by tomv564** [tomv564/LSP#94]
Sublime provides great tools for navigating familiar code, but the outline view in the new Atom-IDE convinced me of the value of seeing a symbol hierarchy of the current document. 
Does Sublime have a good way to present this?

https://blog.atom.io/2017/09/12/announcing-atom-ide.html

**Comment by rwols** [comment]
No, you'll have to go the GitSavvy way and do it all text-based.

**Comment by tomv564** [comment]
There is another Outline package and this CodeMap package (https://forum.sublimetext.com/t/codemap-interactive-code-tree-plugin/26056), but I can foresee much fighting with multi-column layouts, not to mention the effort to get the view itself GitSavvy-like slick.
I'll likely close this issue unless someone produces a reliably managed sidebar view component.
